.github/workflows/npm-dry-run.yml

```diff
@@ -0,0 +1,18 @@
+on:
+  push:
+    branches: ['feature/split-release']
```

Comment:
If there's a simple way to do so, you could probably run the dry-run on any not-`main` branch.

Reply:
If we do, though, I'm not sure how we control which GitHub Actions workflow will run first (`npm-dry-run.yml` vs. `release.yml`).

Reply:
We can run it on non "main" branches with an if statement. I don't think we can specify which workflow order though.

Reply:
we could just run it on develop and that way it will only happen sometime before a release. Otherwise we could try to combine the workflows using a conditional to do `publish` or not and `needs` to make sure dry_run was succesful.

Reply:
what do you think of this approach https://github.com/silinternational/ui-components/pull/250

Reply:
I'll take your suggestion above @forevermatt and merge this, then continue improving on the above PR.